docs/docs/migration-guide.mdx

```diff
@@ -57,34 +57,59 @@ deprecated:
 - [Two-Stage-Fallback Policy](./policies.mdx#two-stage-fallback-policy)
 - [Form Policy](./policies.mdx#form-policy)
 
-#### Migrating from the Mapping Policy
+:::tip Automatic migration
+Rasa Open Source can take care of automatically migrating the Mapping Policy,
+the Fallback Policy and the Two-Stage-Fallback Policy for you. To do so, run the
+following command:
+
+```bash
+rasa data convert config
+```
+
+This command will take care of updating your `config.yml`, `domain.yml` and `rules.yml`, while
+making backups of your pre-existing files using the `.bak` suffix.
+
+You can follow the steps below if you wish to do this migration manually.
+Please note that you would still need to [migrate the Form Policy](#migrating-from-the-form-policy) manually.
+:::
+
+#### Manually migrating from the Mapping Policy
 
 If you previously used the [Mapping Policy](./policies.mdx#mapping-policy), you
 can follow the documentation on [FAQs](./chitchat-faqs.mdx) to convert your mapped
 intents to rules. Suppose you previously mapped an intent `ask_is_bot` as follows:
 
-```yaml-rasa
+```yaml-rasa title="domain.yml"
 intents:
  - ask_is_bot:
      triggers: action_is_bot
 ```
 
 This becomes the following rule:
 
-```yaml-rasa
+```yaml-rasa title="rules.yml"
 rules:
 - rule: Rule to map `ask_is_bot` intent
   steps:
   - intent: ask_is_bot
   - action: action_is_bot
 ```
 
-#### Migrating from the Fallback Policy
+After removing the triggers from your domain file, you can finally replace the
+Mapping Policy with the [Rule Policy](./policies.mdx#rule-policy) in your model configuration:
```

Comment:
should we give an example of how the domain looks without `triggers:`?

Reply:
yep why not